packages/components/spacings/stack/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@commercetools-uikit/spacings-stack",
```

Comment:
Hmm is there a particular reason for having the spacings components split?

Reply:
treeshaking 😄

Reply:
otherwise you end up with too many static properties to treeshake right?

Reply:
Yeah ok. Should we do the same for the typography `Text` components? 🤔

Reply:
hm, yeah we could.